Plots/Trajectory/NCL_polyg_14.py

```diff
@@ -0,0 +1,96 @@
+"""
+NCL_polyg_14.py
+================
+This script illustrates the following concepts:
+    - Drawing polylines and markers using great circle paths
+    - Using gc_latlon to calculate a great circle path
+    - Attaching polylines and markers to a map plot
+    
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/polyg_14.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/polyg_14_2_lg.png
+                         https://www.ncl.ucar.edu/Applications/Images/polyg_14_1_lg.png
+"""
+
+
+###############################################################################
+# Import packages:
+import numpy as np
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import cartopy.feature as cfeature
+from geographiclib.geodesic import Geodesic
+
+from geocat.viz import util as gvutil
+
+###############################################################################
+#Plot
+
+def Plot(color, ext, xext, yext, title, style, pt):
```

Comment:
I recommend adding a comment explaining what each argument is and what this function does.

Reply:
Added in most recent push